README.md

```diff
@@ -97,6 +97,7 @@ These steps are to be completed by the author of the blog post.
    - Merge the PR
    - Once the site rebuilds, check that everything is correct on the [blogs-draft site](https://blogs-draft-openlibertyio.mybluemix.net/blog/) or [draft site](https://draft-openlibertyio.mybluemix.net/blog/).
 10. Get reviewers to review the updates in your new PR.
+11. If you haven't already, create an account on DZone.com with your display name, username, and an email address, then send those details to @lauracowen. This gives DZone.com the option to syndicate our blog posts (they can only syndicate posts for authors who have DZone.com accounts).
```

Comment:
Ah probably better to put my email address. Though what's the best way to do that so I don't get totally spammed?

Reply:
Good point about email bots/spammers. I don't have a suggestion how to obfuscate email addresses to avoid them.

Let's start with adding "on Slack" for now. When we need to collaborate with a non-IBMer, then we can improve the instructions then. Is that OK?